raven/utilities/gis.py

```diff
@@ -380,13 +401,17 @@ def get_hydrobasins_attributes_wfs(
                 outputFormat="json",
                 filter=filterxml,
             )
+            q = Request(method=method, url=GEO_URL, params=params).prepare().url
 
-            q = Request("GET", GEO_URL, params=params).prepare().url
+            # NOTE: This could be rewritten for OWSLib to return the dataset instead of the URL if the need arises
+            # from owslib.wfs import WebFeatureService
+            # wfs = WebFeatureService(url=GEO_URL, version="1.1.0", timeout=30)
+            # req = wfs.getfeature(**params)
```

Comment:
?

Reply:
Oh, this was just a thought. I can remove this.